test/integration/master_restart_test.lua

```diff
@@ -0,0 +1,83 @@
+local fio = require('fio')
+local t = require('luatest')
+local g = t.group()
+
+local helpers = require('test.helper')
```

Comment:
Please, use modern style:

```lua
local t = require('luatest')
local h = require('test.helper')
local g = t.group()
```

Reply:
See, for example, https://github.com/tarantool/cartridge/blob/master/test/integration/restart_replication_test.lua